ingest/api/std_name_units.json

```diff
@@ -0,0 +1,424 @@
+        "unit": "kJ/m2",
+        "alias": [
+            "kJm-2",
+            "kJ m-2"
+        ],
+        "symbol": "kJ/m2"
+    },
+    "surface_snow_thickness": {
+        "unit": "cm",
+        "symbol": "cm"
+    },
+    "surface_temperature": {
+        "unit": "degC",
+        "symbol": "degC",
+        "conversion": {
+            "K": {
+                "add": 272.14
+            }
+        }
+    },
+    "surface_upwelling_longwave_flux_in_air": {
+        "unit": "W/m2",
+        "alias": [
+            "W m-2",
+            "Wm-2"
+        ],
+        "symbol": "W/m2"
+    },
+    "surface_upwelling_radiance_in_air": {
+        "unit": "kJ/m2",
+        "alias": [
+            "kJm-2",
+            "kJ m-2"
+        ],
+        "symbol": "kJ/m2"
+    },
+    "surface_upwelling_shortwave_flux_in_air": {
+        "unit": "W/m2",
+        "alias": [
+            "W m-2",
+            "Wm-2"
+        ],
+        "symbol": "W/m2"
+    },
+    "tendency_of_air_pressure": {
+        "unit": "hPa",
+        "symbol": "hPa",
+        "alias": [],
+        "conversion": {
+            "Pa": {
+                "mul": 0.01
+            }
+        }
+    },
+    "tendency_of_surface_air_pressure": {
+        "unit": "hPa",
+        "symbol": "hPa",
+        "conversion": {
+            "Pa": {
+                "mul": 0.01
+            }
+        }
+    },
+    "thickness_of_snowfall_amount": {
+        "unit": "cm",
+        "symbol": "cm"
+    },
+    "ultraviolet_index": {
+        "unit": "index",
+        "symbol": "index"
+    },
+    "virtual_temperature": {
+        "unit": "degC",
+        "symbol": "degC",
+        "conversion": {
+            "K": {
+                "add": 272.14
+            }
+        }
+    },
+    "visibility_in_air": {
+        "unit": "m",
+        "symbol": "m"
+    },
+    "wet_bulb_temperature": {
+        "unit": "degC",
+        "symbol": "degC",
+        "conversion": {
+            "K": {
+                "add": 272.14
+            }
+        }
+    },
+    "wind_from_direction": {
+        "unit": "degrees",
+        "symbol": "degrees"
+    },
+    "wind_speed": {
+        "unit": "m/s",
+        "alias": [
+            "m s-1",
+            "ms-1"
+        ],
+        "symbol": "m/s"
+    },
+    "wind_speed_of_gust": {
+        "unit": "m/s",
+        "alias": [
+            "m s-1",
+            "ms-1"
+        ],
+        "symbol": "m/s"
+    },
+    "air_pressure": {
+        "unit": "hPa",
+        "symbol": "hPa",
+        "alias": [],
+        "conversion": {
+            "Pa": {
+                "mul": 0.01
+            }
+        }
+    },
+    "air_pressure_at_mean_sea_level": {
+        "unit": "hPa",
+        "symbol": "hPa",
+        "alias": [],
+        "conversion": {
+            "Pa": {
+                "mul": 0.01
+            }
+        }
+    },
+    "air_temperature": {
+        "unit": "degC",
+        "conversion": {
+            "K": {
+                "add": 272.14
```

Comment:
Pretty sure that:
Celcius = K - 273.15

So it should be `273.15` or `-273.15`, depending on which direction the code works.. Anyway, 0 C should come out as 273.15 K.

This is used in many places in this file.

Reply:
It should indeed be `-273.15`, or it would be rather hot from the places reporting kelvins. Should be fixed now.

Reply:
You still have `-273.14` in the code, should be `-273.15`.

Reply:
One digit can be enough. See [Table 1](https://library.wmo.int/viewer/68695/download?file=8_I-2023_en.pdf&type=pdf&navigator=1) , page 103.

Reply:
The table is about measurement accuracy. I would say the conversion has to be as accurate as possible (-273.15 is exact). Because if the original measurement has 2 decimal places, the conversion would otherwise be wrong.

Reply:
I don't think we currently control output number accuracy in the API. We give back whatever the original number that was put into the datastore by the ingestor.

Reply:
This is an [example](https://api.esoh.met.no/collections/observations/locations/0-578-0-46800?parameter-name=air_temperature%3A%2A%3A%2A%3A%2A&datetime=2024-06-21T03%3A00%3A00Z&f=CoverageJSON) :

`  "ranges": {
    "air_temperature:0.0:point:PT0S": {
      "type": "NdArray",
      "dataType": "float",
      "axisNames": [
        "t",
        "y",
        "x"
      ],
      "shape": [
        1,
        1,
        1
      ],
      "values": [
        9.19999981
      ]
    }
  }
`
Round to two digits?

Reply:
I have updated the code to keep as many decimals as the original measurement.